smilerating/src/main/java/com/hsalf/smilerating/SmileRating.java

```diff
@@ -311,31 +317,33 @@ protected void onDraw(Canvas canvas) {
-                                (mSelectedSmile == TERRIBLE || mPreviousSmile == TERRIBLE) ? mAngryColor : mNormalColor));
-                mScaleMatrix.reset();
-                mSmilePath.computeBounds(mScaleRect, true);
-                float nonSelectedScale = mFloatEvaluator.evaluate(
-                        mInterpolator.getInterpolation(mMainSmileyTransformaFraction), getScale(NONE), 1f);
-                mScaleMatrix.setScale(nonSelectedScale, nonSelectedScale,
-                        mScaleRect.centerX(), mScaleRect.centerY());
-                mDummyDrawPah.reset();
-                mDummyDrawPah.addPath(mSmilePath, mScaleMatrix);
-
-                canvas.drawCircle(mFaceCenter.x, mFaceCenter.y,
-                        nonSelectedScale * (mHeight / 2f), mBackgroundPaint);
-                canvas.drawPath(mDummyDrawPah, mPathPaint);
-            } else {
-                canvas.drawCircle(mFaceCenter.x, mFaceCenter.y, mHeight / 2f, mBackgroundPaint);
-                canvas.drawPath(mSmilePath, mPathPaint);
-            }
+//            if (mSmileyNotSelectedPreviously) {
+//                Log.i(TAG, "Non selection");
+//                /*mPathPaint.setAlpha(Math.round(255 * mMainSmileyTransformaFraction));
+//                mBackgroundPaint.setAlpha(Math.round(255 * mMainSmileyTransformaFraction));*/
+//                mPathPaint.setColor((Integer) mColorEvaluator
+//                        .evaluate(mMainSmileyTransformaFraction, mPlaceHolderFacePaint.getColor(), mDrawingColor));
+//                mBackgroundPaint.setColor((Integer) mColorEvaluator
+//                        .evaluate(mMainSmileyTransformaFraction, mPlaceHolderCirclePaint.getColor(),
+//                                (mSelectedSmile == TERRIBLE || mPreviousSmile == TERRIBLE) ? mAngryColor : mNormalColor));
+//                mScaleMatrix.reset();
+//                mSmilePath.computeBounds(mScaleRect, true);
+//                float nonSelectedScale = mFloatEvaluator.evaluate(
+//                        mInterpolator.getInterpolation(mMainSmileyTransformaFraction), getScale(NONE), 1f);
+//                mScaleMatrix.setScale(nonSelectedScale, nonSelectedScale,
+//                        mScaleRect.centerX(), mScaleRect.centerY());
+//                mDummyDrawPah.reset();
+//                mDummyDrawPah.addPath(mSmilePath, mScaleMatrix);
+//
+//                canvas.drawCircle(mFaceCenter.x, mFaceCenter.y,
+//                        nonSelectedScale * (mHeight / 2f), mBackgroundPaint);
+//                canvas.drawPath(mDummyDrawPah, mPathPaint);
+//            } else {
+//                canvas.drawCircle(mFaceCenter.x, mFaceCenter.y, mHeight / 2f, mBackgroundPaint);
+//                canvas.drawPath(mSmilePath, mPathPaint);
+//            }
+            canvas.drawCircle(mFaceCenter.x, mFaceCenter.y, mHeight / 2f, mBackgroundPaint);
+            canvas.drawPath(mSmilePath, mPathPaint);
         }
     }
 
```

Comment:
Should probably add option to set XML attributes for each of the smiley colors as well as functions setBadSmiley color.